flow/probes/ebpf.go

```diff
@@ -156,6 +156,14 @@ func (p *EBPFProbe) flowFromEBPF(ebpfFlow *EBPFFlow, updatedAt int64) *flow.Flow
 				A:        portA,
 				B:        portB,
 			}
+			f.TCPMetric = &flow.TCPMetric{
+				ABSynStart: int64(kernFlow.transport_layer.ab_syn),
```

Comment:
timestamp won't be correct regarding `start` and `last` of the flow because it is a monotonic clock. Please check line 338 to see how to get the real one.

Reply:
4) Can u please provide me with a few more details? What do you mean "a monotonic clock" and why won't this work..
Thx

Reply:
The timestamp that you get from the kernel is different from the time that you get from userspace. It is the time since some unspecified starting point not the real time.